src/shared/seccomp-util.c

```diff
@@ -10,7 +10,6 @@
 #include <sys/stat.h>
 
 /* include missing_syscall_def.h earlier to make __SNR_foo mapped to __NR_foo. */
```

Comment:
Is it safe to move the header now??

Reply:
Probably not, disabled clang-format for this specific bit to avoid it accidentally moving in the future.